test/functional/api/v2/job_invocations_controller_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'test_plugin_helper'
```

Comment:
I'm not sure i fully understand what you mean by this comment?  This was modeled after the existing job_templates_controller api controller